MODULE.bazel

```diff
@@ -51,6 +51,8 @@ single_version_override(
     ],
 )
 
+bazel_dep(name = "abseil-cpp", version = "20240722.0", repo_name = "com_google_absl")
```

Comment:
could you please remove it from `WORKSPACE.bzlmod`? 

It's what our bzlmod setup is using today and I think using MODULE.bazel is much better.